stories/coverage.fmf

```diff
@@ -0,0 +1,100 @@
+story:
+    As a QE I want to document the test coverage,
+    so that it can be easily tracked which requirement
+    is covered by which test case and vice-versa.
+
+/notes:
+    summary:
+        Story notes
+    story:
+        As a product owner I want to add additional content to
+        the requirements, like notes or context,
+        so that extra information can be provided.
+    description:
+        There should be a parameter that has no strict
+        requirement and can be used to provide any
+        extra information that may be necessary.
+    example: |
+        story:  As an admin I want product A doing B,
+                so that I can use it for C.
+        notes:  The feature should help our customer
+                to achieve D.
+    priority: low
```

Comment:
You are right, the description attribute pretty much covers this. Let me drop it. It can be re-opened when someone comes with a strong argument. ATM this is probably not needed.

Reply:
Yes, agreed, thanks.